docs/index.md

```diff
@@ -1,36 +1,62 @@
 ---
 layout: ""
-page_title: "Provider: kentik-cloudexport"
+page_title: "Provider: Kentik CloudExport"
 description: |-
-  The kentik-cloudexport provider provides resources to interact with Kentik CloudExport API.
+  The Kentik CloudExport provider provides resources to interact with Kentik CloudExport API.
 ---
 
-# kentik-cloudexport provider
+# Kentik CloudExport Provider
 
-The kentik-cloudexport provider provides resources to interact with Kentik CloudExport API.
+The Kentik CloudExport provider provides resources to interact with Kentik CloudExport API.
 
-Interacting with Kentik CloudExport API requires authorization with valid (email, token) pair.
+## Example usage
 
-## Example Usage
+Go to folder with Terraform `.tf` definition files for CloudExport resources/data sources ([./examples/**](./examples)):
```

Comment:
Please use "Cloud Export" instead of the ugly CamelCase. Applies to all occurrences.

Reply:
Done